translations/messages.fr.xlf

```diff
@@ -0,0 +1,3063 @@
+      <trans-unit id="75731290119916717" datatype="html">
+        <source>A stack must always be moved by as many spaces as there are pieces in the stack.</source>
+        <target>La distance effectuée par le mouvement doit correspondre à la taille de la pile de pièces.</target>
+      </trans-unit>
+      <trans-unit id="8101145555087657570" datatype="html">
+        <source>The stack must land on an occupied space.</source>
+        <target>Le déplacement doit se terminer sur une case occupée.</target>
+      </trans-unit>
+      <trans-unit id="5010267418211867946" datatype="html">
+        <source>Moving</source>
+        <target>Déplacement</target>
+      </trans-unit>
+      <trans-unit id="364149588471541692" datatype="html">
+        <source>At Dvonn, each hexagonal space contains a stack of pieces.
+        If no number is indicated on a stack, it means that it only contains one piece.
+        The number written on a stack indicates the number of pieces within that stack, hence the number of points its owner gets.
+        The owner of a stack is the one that has a piece at the top of the stack.
+        Only the owner of a stack can move it.
+        A stack cannot be moved if it is surrounded by 6 other stacks.
+        A stack is moved by as many spaces as there are pieces within, in a straight line, and it should land on an occupied space.
+        This line cannot move along the vertex of two neighboring spaces, as would a vertical move.
+        Therefore, there are six possible directions.
+        The player with dark pieces starts.&lt;br/&gt;&lt;br/&gt;
+        You&apos;re playing Dark, click on a stack and move it by one space.</source>
+        <target>Au Dvonn, chaque case hexagonale comporte une pile de pièce.
+        Si aucun nombre n&apos;est indiqué sur une pile, c&apos;est qu&apos;elle ne comporte qu&apos;une pièce.
+        Le nombre écrit sur une pile correspond au nombre de pièces empilées et donc le nombre de point qu’elle rapporte à son propriétaire.
+        Son propriétaire est celui dont une pièce est au sommet de la pile.
+        Seul son propriétaire peut déplacer la pile.
+        Il ne peut pas la déplacer si elle est entourée par 6 autres piles.
+        Il la déplace d’autant de cases que sa hauteur, en ligne droite, et doit atterrir sur une case occupée.
+        Cette ligne droite ne peut pas passer le long de l&apos;arête de deux cases voisines, comme le ferait un déplacement vertical.
+        Il y a donc six directions possibles.
+        Le joueur avec les piles foncées commence.&lt;br/&gt;&lt;br/&gt;
+        Vous jouez avec Foncé, cliquez sur une pile puis déplacez la d&apos;une case.</target>
+      </trans-unit>
+      <trans-unit id="8769382369391878948" datatype="html">
+        <source>Disconnection</source>
+        <target>Déconnection</target>
+      </trans-unit>
+      <trans-unit id="7296336707589349573" datatype="html">
+        <source>Red pieces are called &quot;sources&quot;.
```

Comment:
there's no red piece on that board!
ps: don't change the color of the piece!!

Reply:
très juste, je mets "pieces with a lightning strike" / "les pièces avec un éclair", au cas où la couleur change dans le futur !

Reply:
en anglais c'est serait pas juste "a lightning" ?
"Lightning strike" semble être "coup de foudre" et pas "un éclair".

AH NON! GOOGLE A DIT QU'IL FAUT METTRE LOVE AT FIRST SIGHT.